dashboard/config/routes.rb

```diff
@@ -179,6 +179,7 @@ def redirect_to_teacher_dashboard
 
   # one-off internal reports
   get '/admin/brook/csppd', to: 'reports#csp_pd_responses', as: 'csp_pd_responses'
+  get '/admin/roxanne/hoc_signups', to: 'admin_reports#hoc_signups', as: 'hoc_signups'
```

Comment:
can we reverse the trend of prefixing these routes with the name of the requester? Event if they're intended to be temporary views, I'd prever to have something like `/admin/temp/` and have a comment in the controller explaining the nature and source of the request.